KIPs/kip-12.md

```diff
@@ -0,0 +1,67 @@
+---
+kip: 12
+title: KyberDMM Expansion and Liquidity Mining on BSC
+status: Active
+type: Mint KNC
```

Comment:
The type of this KIP is not `Mint KNC` as we are not supposed to mint more KNC, we spend KNC from the short executor. Lets change the type to `Spend KNC` instead.

Reply:
Noted